java/app/src/main/java/com/google/android/gms/samples/wallet/activity/CheckoutActivity.java

```diff
@@ -342,6 +412,56 @@ public void requestPayment(View view) {
     }
   }
 
+  /**
+   * Calls
+   * {@link PaymentsClient#getPaymentCardRecognitionIntent(PaymentCardRecognitionIntentRequest)} API
+   * and fetches the {@link PendingIntent} needed to launch the payment card recognition
+   * {@code Activity}. Sets the "scan card" button to visible if the call is successful.
+   */
+  public void requestPaymentCardOcrIntent() {
+    PaymentCardRecognitionIntentRequest request =
+        PaymentCardRecognitionIntentRequest.getDefaultInstance();
+    paymentsClient
+        .getPaymentCardRecognitionIntent(request)
```

Comment:
Would it be possible to make this an optional parameter, that when not included, the default instance above is used (and hence the need for lines 422-423 is not required)?

Reply:
I think it was a conscious decision to keep the request parameter. I added some comments. Hopefully it clears up a bit for the developers.